docs/setup/installation.md

```diff
@@ -55,6 +55,21 @@ then the `git clone` command did not run correctly.
 Make sure you have git installed and run it again
 with `--verbose` to give more logging output.
 
+If you see
+
+```bash
+fatal: destination path 'commcare-cloud' already exists and is not an empty directory.
```

Comment:
Just kind of curious, I've never seen this. When does/can it happen?

Reply:
```sh
$ mkdir commcare-cloud
$ touch commcare-cloud/.git
$ git clone git@github.com:dimagi/commcare-cloud.git
fatal: destination path 'commcare-cloud' already exists and is not an empty directory.
```